tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers:
+#   proxy-httpd
+#   proxy-salt-broker
+#   proxy-squid
+#   proxy-ssh
+#   proxy-tftpd
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    assert_script_run("podman pull $image");
```

Comment:
Don't you want to run a command to start a container from that image? Not 100% needed, just suggesting to at least check that the image can be started :)

Reply:
Yes I was gonna do it next, just not sure what command to run to have some kind of out put.... would something as basic as `podman run $image podman run  /etc/os-release` be enough?

Reply:
Yes that would be enough :)

Reply:
I added it, how does openQA validate it? Through a needle?

Reply:
No, it expects that the return value of the command is 0. If not, it will fail.